docs/_macros/compare.md

```diff
@@ -1,20 +1,10 @@
 ---
 layout: default
 class: Macro
-title: compare NUMBER NUMBER
-summary: Compare two numbers. 0 is equal, 1 means a > b, -1 is a < b.
+title: compare STRING STRING
+summary: Compare two strings by using the compareTo method of the String class. <astring>.compareTo(<bstring>)
 ---
 
-	static String	_ncompare	= "${ncompare;<anumber>;<bnumber>}";
-
-	public int _ncompare(String args[]) throws Exception {
-		verifyCommand(args, _ncompare, null, 3, 3);
-		double a = Double.parseDouble(args[1]);
-		double b = Double.parseDouble(args[2]);
-		if (a > b)
-			return 1;
-		if (a < b)
-			return -1;
-		return 0;
-	}
+Compare two strings by using the compareTo method of the String class. <astring>.compareTo(<bstring>). 0 is equal, 1 means <astring> > <bstring>, -1 is <astring> < <bstring>.
 
+	${compare;"valueA";"valueB"}
```

Comment:
You need to remove the double quotes in the example. 

`${compare;valueA;valueB}`